source/YearlyOverview/YearlyJS.js

```diff
@@ -114,6 +114,24 @@ function renderGoals(goals) {
     });
 }
 
+/**
+ * check if the form is empty. If it is, return true. If not, return false.
+ * @returns whether the form is valid or not
+ */
+function checkForm() {
+    console.log('DEBUG: checkForm()');
+    if (document.querySelector('.entry-form-text').value === '') {
+        console.log('DEBUG: form is empty');
+        alert('Please enter a goal');
```

Comment:
Please remove the console.logs. 
Also, write an explanation in the comments for why it needs to return true or false since those values don't seem to be used, only alerting or not seems to be actively used. I'm not sure if they are used in onclick="return checkform()".
Are they more for the developer to see?